python/ray/rllib/examples/carla/train_ppo.py

```diff
@@ -28,14 +28,14 @@
     "carla": {
         "run": "PPO",
         "env": "carla_env",
-        "trial_resources": {"cpu": 4, "gpu": 1},
```

Comment:
why does Carla no longer need GPU specification?

Reply:
It's auto-detected for rllib since 0.5.0.